assets/scripts/main.js

```diff
@@ -5,14 +5,23 @@
    * If the page takes time to be loaded, the user could click multiple
    * times on the place order button and this would generate multiple orders.
    * To prevent this we stop form submit event propagation after the first click.
+   *
+   * Covers the case of woocommerce-german-market second checkout page being
+   * disabled. If form fields validation fails, the order submit button is
+   * enabled again to allow the user to resend the form.
    */
-  $('form.checkout.wgm-second-checkout').submit((e) => {
-    if ($('#place_order').hasClass('disabled')) {
+
+  $('#place_order').click(function disableSubmitOrder(e) {
```

Comment:
It's possible that the unspecified selector might conflict with the multi-step-checkout that was built for STIM.  However, as the website did not go live, we may ignore that possible problem and wait for the case to reappear.

Reply:
@juanlopez4691 — STIM is not live and `shop-standards` is not installed since it was created before STIM. Please, just ignore STIM case.

Reply:
@juanlopez4691 — STIM is not live and `shop-standards` plugin is not installed there. Please, proceed ignoring STIM case.